react/src/components/buttons/SprkButton.js

```diff
@@ -73,11 +79,18 @@ SprkButton.propTypes = {
   analyticsString: PropTypes.string,
   /** Content to render inside of the SprkButton */
   children: PropTypes.node,
+  // TODO: Deprecate on next release
```

Comment:
Hey @RVMendoza can you sync with Steph and get an issue in for the deprecate work and put the issue number here in the comments?

Reply:
@afebbraro this is finished and connected to issue that removes on next release https://github.com/sparkdesignsystem/spark-design-system/issues/3557